### DISCUSSION TITLE
Query on Conversion code in using them for other wavelengths

### DISCUSSION THREAD

**Discussion by SarathSS98** [mboquien/cigale#34] [General]
Hello all,

I have a query in the conversion code where I have a sample of sources which have their fluxes measured in erg/s/cm2/A while the code works for erg/s/cm2. To compensate this multiplying the fluxes by the rest frame wavelength can help . I face this with 1450A observation. I do not understand how can I interpret the Elo and Eup parameters where for x-ray ranges they have in keV. Any ideas on using this conversion code for other wavebands (1450A in my case) is much appreciated.

Thanks,
Sarath S S

**Comment by Guang91** [comment]
The code is for X-ray bands. 1450A is UV not X-ray. So you shouldn't use it. 
You should convert your flux to mJy and use the correct filter.
